<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stats</class>
 <widget class="QMainWindow" name="stats">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stats Chart</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QChartView" name="chartYear">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>631</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QChartView" name="chartMonth">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>631</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QChartView" name="chartDay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>801</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReset">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>730</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblYear">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Yearly Chart</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMonth">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Monthly  Chart</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDaily">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Daily  Chart</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboYear_Month">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>260</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblSelectYear_Month">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>260</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Year:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSelectYear_Day">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>500</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Year:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboYear_Day">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>500</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboMonth_Day">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>500</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblSelectMonth_Day">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>500</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Month:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCostThisMonth">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>310</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Totaly cost at this month:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editCostThisMonth">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>330</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="editCostLastMonth">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>380</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblCostLastMonth">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>360</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Totaly cost at last month:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editMonthSaving">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>430</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblSavingMonth">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>410</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Savings (USD):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editCostLastYear">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>140</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="editYearSaving">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>190</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblSavingYear">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>170</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Savings (USD):</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCostLastYear">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>120</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Totaly cost at last year:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editCostThisYear">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>90</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblCostThisYear">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>70</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Totaly cost at this year:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
